pkg/generators/column.go

```diff
@@ -32,3 +38,20 @@ func CreateIndexesForColumn(table *typedef.Table, maxIndexes int) []typedef.Inde
 	}
 	return indexes
 }
+
+func AddReferencesForIndexes(table *typedef.Table) {
```

Comment:
Please move it into `typedef`
```suggestion
func (t *Table) LinkIndexAndColumns() error {
```

Reply:
Done